src/test/java/uk/gov/hmcts/divorce/citizen/event/CitizenUpdateCaseStateAatTest.java

```diff
@@ -73,4 +75,41 @@ void shouldNotUpdateCaseStateWhenStateIsInvalid() {
         assertThat(response.getState()).isEqualTo(originalState);
         assertThat(response.getData().getApplicant2().getMiddleName()).isNotEmpty();
     }
+
+    @Test
+    void shouldUpdateApplicant1MiddleNameToEmptyStringWhenChangingState() {
+        final long caseId = 1L;
+        final CaseDetails<CaseData, State> caseDetails = new CaseDetails<>();
+        final CaseData caseData = CaseData.builder().build();
+
+        caseData.setApplicant2(new Applicant());
+        caseData.getApplicant2().setMiddleName("Holding");
+
+        caseDetails.setData(caseData);
+        caseDetails.setId(caseId);
+
+        final AboutToStartOrSubmitResponse<CaseData, State> response = citizenUpdateCaseStateAat.aboutToSubmit(caseDetails, caseDetails);
+
+        assertThat(response.getData().getApplicant2().getMiddleName()).isEqualTo(EMPTY_STRING);
```

Comment:
you can use assertThat(response.getData().getApplicant2().getMiddleName()).isEmpty() ?

Reply:
done thanks